packages/lib/src/components/Giftcard/Giftcard.tsx

```diff
@@ -29,8 +29,7 @@ export class GiftcardElement extends UIElement {
             paymentMethod: {
                 type: this.constructor['type'],
                 brand: this.props.brand,
-                encryptedCardNumber: this.state.data?.encryptedCardNumber,
-                encryptedSecurityCode: this.state.data?.encryptedSecurityCode
+                ...this.state.data
```

Comment:
We might want to conditionally add the `encryptedExpiryDate` (and leave the previous ones) to prevent exposing incorrect data in case anything ends up accidentally in state.data in the future.

Reply:
Well we shouldn't be _accidentally_ doing anything! (and this is how the `Card` comp works).

But if you do want to be more specific then it would be `encryptedExpiryMonth` & `encryptedExpiryYear` - since securedFields breaks the date down into these 2 properties